icu4j/main/core/src/main/java/com/ibm/icu/impl/JavaTimeConverters.java

```diff
@@ -253,6 +257,69 @@ public static TimeZone zoneOffsetToTimeZone(ZoneOffset zoneOffset) {
         return new SimpleTimeZone(zoneOffset.getTotalSeconds() * 1_000, zoneOffset.getId());
     }
 
+    /**
+     * Converts a {@link DayOfWeek} to a {@link Calendar}.
+     *
+     * <p>
+     * This method creates a {@link Calendar} instance that represents
+     * a day that is the same day of week as specified by {@link DayOfWeek}.
+     * It is set somewhere close to epoch time.
+     *
+     * <p>
+     * <b>Note:</b> this should only be used to format it using a pattern or skeleton
+     * with a day of week field only.<br>
+     * That means that {@code c}-{@code cccccc} patterns are recommended, {@code E}-{@code EEEEEE}
+     * and {@code e}-{@code eeeeee} are likely wrong (because they are not stand-alone).
+     * Anything else is clearly wrong.<br>
+     * It does not make sense to format a {@code DayOfWeek} as {@code "MMMM d, y"}.<br>
+     * See {@link https://unicode.org/reports/tr35/tr35-dates.html#dfst-weekday}.
+     *
+     * @param dow The {@link DayOfWeek} to convert.
+     * @return A {@link Calendar} instance representing the same day of week
+     *         as the one specified by the input.
+     *
+     * @deprecated This API is ICU internal only.
+     */
+    @Deprecated
+    @SuppressWarnings("JavaTimeDefaultTimeZone")
+    public static Calendar dayOfWeekToCalendar(DayOfWeek dow) {
+        return millisToCalendar(dayOfWeekToMillis(dow));
+    }
+
+    /**
+     * Converts a {@link Month} to a {@link Calendar}.
+     *
+     * <p>
+     * This method creates a {@link Calendar} instance that represents
+     * the same month as specified by {@link Month}.
+     * It is set somewhere close to epoch time.
+     *
+     * <p>
+     * <b>Note:</b> this should only be used to format it using a pattern or skeleton
+     * with a day of month field only.<br>
+     * That means that {@code L}-{@code LLLLL} patterns are recommended, {@code E}-{@code MMMMM}
+     * is likely wrong (because it is not stand-alone). Anything else is clearly wrong.<br>
+     * It does not make sense to format a {@code Month} as {@code "MMMM d, y"}.<br>
```

Comment:
```suggestion
     * with a day of month field only.
     * That means that {@code L}-{@code LLLLL} patterns are recommended, {@code E}-{@code MMMMM}
     * is likely wrong (because it is not stand-alone). Anything else is clearly wrong.
     * It does not make sense to format a {@code Month} as {@code "MMMM d, y"}.
```